controller/lib/core/blower_fsm.h

```diff
@@ -162,8 +168,11 @@ class BlowerFsm {
   BlowerSystemState DesiredState(Time now, const VentParams &params) {
     // Immediately turn off the ventilator if params.mode == OFF; otherwise,
     // wait until the end of a cycle before implementing the mode change.
-    if (params.mode == VentMode_OFF ||
+    bool is_new_breath = false;
+    if ((params.mode == VentMode_OFF &&
+         !std::holds_alternative<OffFsm>(fsm_)) ||
         std::visit([&](auto &fsm) { return fsm.finished(now); }, fsm_)) {
+      is_new_breath = true;
```

Comment:
Please comment why it's appropriate/necessary to set this to true when the vent transitions from on to off